src/renderers/shaders/ShaderChunk/lights_spherelambert_pars_fragment.glsl.js

```diff
@@ -0,0 +1,79 @@
+export default /* glsl */`
+varying vec3 vViewPosition;
+
+struct LambertMaterial {
+
+	vec3 diffuseColor;
+	float specularStrength;
+
+};
+float safeacos(const float x) {
+    return acos(clamp(x, -1.0, 1.0));
+}
+
+float phase(float u) {
+    return (2.0*(sqrt(1.0 - u*u) - u*acos(u)))/(3.0*PI*PI);
+}
+vec3 shadeLambertianSphereBRDF(vec3 wi, vec3 wo, vec3 norm,vec3 kd) { 
+
+    float ui = dot(wi, norm);
+    float uo = dot(wo, norm);
+    if (ui < 0.0 || uo < 0.0) return vec3(0.0);
+    
+	float uIuO = ui*uo;
+	vec3 c =  (1.0 - pow(1.0 - kd, vec3(2.73556))) / (1.0 - 0.184096*pow(1.0 - kd, vec3(2.48423)));
+	float minusiodot = -dot(wi, wo);
+    float ui2 = ui*ui;
+    float uo2 = uo*uo;
+    float S = sqrt((1.0-ui2)*(1.0-uo2));
+    float cp = -((minusiodot + uIuO)/S);
+    float phi = safeacos(cp);
+    
+    
+    // Single-Scattering component, corresponds to "f_1" in the paper.
+    vec3 SS = c*(phase(minusiodot) / (ui + uo));
+    
+    // The  block is a literal coding of Equation 48 from the paper.
+
+	vec3 fr = max( 
+        vec3(0.0), 
+        SS + 0.234459*pow(kd, vec3(1.85432)) \
+           + (0.0151829*(c-0.24998)*(abs(phi)+sqrt(uIuO))) / (0.113706 + (safeacos(S)/S))
+    );
+    return PI * uo * fr;
+    
+}
+/* struct IncidentLight {
+	vec3 color;
```

Comment:
The color and intensity of the incident light is missing from this implementation.

Reply:
This is fixed by modifying the implementation to support the incident light color and intensity. It is not fixed by removing a comment.